.devcontainer/Dockerfile

```diff
@@ -1,3 +1,15 @@
-FROM node:18-bullseye
+FROM ubuntu
```

Comment:
What about specifying the version to prevent it from potentially break in the future?

Reply:
Will use 24.04

Reply:
Shouldn't we have something even more precise?

at CodinGame we even reference images with their hashes now

Reply:
What about noble-20240605. That is the current tag of 24.04 LTS. That will not change. 24.04 will move forward with fixes to the LTS version.

Reply:
Done ⬇️